packages/react-jsx-highcharts/src/components/UseManualEventHandlers/index.js

```diff
@@ -0,0 +1,30 @@
+import useHighcharts from '../UseHighcharts';
+import usePrevious from '../UsePrevious';
+import { getEventsConfig } from '../../utils/events';
+import getModifiedProps from '../../utils/getModifiedProps';
+
+const useManualEventHandlers = function(props, target) {
+  const Highcharts = useHighcharts();
+  const eventHandlers = getEventsConfig(props);
+  const previousEventHandlers = usePrevious(eventHandlers);
+
+  const modifiedEvenHandlers = getModifiedProps(
+    previousEventHandlers,
+    eventHandlers
+  );
+
+  if (modifiedEvenHandlers !== false) {
+    Object.entries(modifiedEvenHandlers).forEach(([eventName, newHandler]) => {
```

Comment:
Think we should use Lodash [`toPairs`](https://lodash.com/docs/4.17.15#toPairs) as Edge 12 & 13 doesn't support this.

Reply:
v.4 was requiring polyfills anyway, but I can rewrite that with Object.keys too.